influunt-api/modules/influunt-core/app/utils/TipoLogControlador.java

```diff
@@ -0,0 +1,14 @@
+package utils;
+
+/**
+ *
+ * ATENÇÃO !!!!
+ *
+ * QUALQUER ALTERAÇÃO NOS TIPOS DEVE SER REFLETIDA NA BUSCA DE LOGS NA WEB.
```

Comment:
Já que é importante, poderia colocar aqui que se esse enum for alterado, deve ser alterado no APP o I18n e as constantes TIPO_LOG.